h/security/role.py

```diff
@@ -0,0 +1,19 @@
+class Role:
+    #: Administrators. These users have super cow powers.
+    Admin = "role:__admin__"
```

Comment:
The value of the strings is arbitrary, but it does appear important that they are strings. I was getting some errors with a real enum.

I took the opportunity to change the names here from `group:__admin__` which is easily confused with the way we express groups: `group:{group.pubid}` especially as one of them is `group:__world__`.

This is an example of the namespace clash between our groups and Pyramid's occasional use of the phrase for principals.